### ISSUE TITLE
Pre training weights

### ISSUE THREAD

**Issue by lyna-tlncgbnn** [ZhichengHuang/CMAE#14]
Thank you very much for your open source code. May I ask if there are weight parameters obtained from the decoder and projector sections during pre training?

**Comment by ZhichengHuang** [comment]
In the pre-training phase, all parameters are randomly initialized.

**Comment by lyna-tlncgbnn** [comment]
> In the pre-training phase, all parameters are randomly initialized.

Yes, what I mean is: Do you have all the after pre trained weight parameters here, not just for the encoder part of downstream tasks。I want to continue pre-training with checkpoint for special dataset。
